### PR TITLE
Update Caranthir Tower Reborn

### PR DESCRIPTION
**Pull request by cecell**
[Badersee SE](https://www.nexusmods.com/skyrimspecialedition/mods/16419) and [Caranthir Tower Reborn](https://www.nexusmods.com/skyrimspecialedition/mods/4269) both make landscape/navmesh changes substantial enough to need patching but as it says, none exist at this time

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -20995,7 +20995,11 @@ plugins:
 
   - name: 'CaranthirTowerReborn.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/4269/' ]
-    msg: [ *patch3rdParty_SRPC ]
+    msg:
+      - *patch3rdParty_SRPC
```
Comment (sibir-ine) [inline masterlist.yaml]: There's an extra whitespace here:

```suggestion
      - *patch3rdParty_SRPC
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Comment by MacSplody** [comment]
Someone left this comment on the Caranthir Tower Reborn mod page.
We could add a message for that as well, Bloodchill Manor `cceejsse005-cave.esm`.
I can test it out in a bit if you don't have that file.

image

**Comment by cecell** [comment]
Hey very cool. Yes please go ahead. I don't have that file

**Comment by MacSplody** [comment]
> Hey very cool. Yes please go ahead. I don't have that file

Cell Location conflict, not sure how that would be patched.
image

---

Bloodchill Manor location
image

Referenced by
image

---

Caranthir Tower Reborn location
image


Referenced by
image

---

Both add objects to that cell as well.

image

image

---

So it would also need a `patchUnavailable` message.

**Review by sibir-ine — APPROVED** [review]

**Pull request merged**